metadata.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "puppet-rsyslog",
-  "version": "5.1.0",
+  "version": "5.1.1-rc0",
```

Comment:
not sure where this came from. I didn't add/change this at all

Reply:
mhm good question. our `travis_release` rake task bumps the version after a git tag was created. Normally this should have been pushed directly to master.

Reply:
:shrug: Not a big deal as long as it doesn't happen again